### DISCUSSION TITLE
Whenever I try to uncomment the resize mode the config file fails and gives the error on next login. Is the code ok?

### DISCUSSION THREAD

**Discussion by shaurya770** [i3/i3#5769] [Q&A] [answered]
![Uploading RDT_20231114_1756547290364786610515091.jpg …]()
I swear I never touched anything

**Comment by orestisfl** [comment]
Your picture link is not working and in any case, please don't use images to show code and config files, just upload the file itself

**Reply by shaurya770** [reply]
i am really sorry, i didn't know.
[config.zip](https://github.com/i3/i3/files/13352670/config.zip)
 here's the file for reference

**Reply by shaurya770** [reply, marked as answer]
hey umm, sorry for wasting your time. Looks like i wasn't uncommenting the closing bracket, I feel like an idiot.

**Reply by orestisfl** [reply]
Don't worry, no time wasted :)

Easy to make this kind of mistakes
